labgrid/remote/exporter.py

```diff
@@ -478,6 +478,23 @@ def _get_params(self):
 
 exports["NetworkService"] = NetworkServiceExport
 
+@attr.s(eq=False)
+class LXAIOBusNodeExport(ResourceExport):
+    """ResourceExport for LXAIOBusNode devices accessed via the HTTP API"""
+
+    def __attrs_post_init__(self):
+        super().__attrs_post_init__()
+        local_cls_name = self.cls
+        self.data['cls'] = "Network{}".format(self.cls)
+        from ..resource import lxaiobus
```

Comment:
Shouldn't the import happen at the very beginning of the method?

Reply:
This is consistent with the usage in the rest of exporter.py. I am ambiguous to this, I'd keep it like this.

Reply:
Okay,